java/viewshed-geoprocessing/src/main/java/com/esri/arcgisruntime/sample/viewshedgeoprocessing/MainActivity.java

```diff
@@ -73,6 +75,7 @@ protected void onCreate(Bundle savedInstanceState) {
     // location services
     ArcGISRuntimeEnvironment.setApiKey(BuildConfig.API_KEY);
 
+    mLoadingView = findViewById(R.id.loadingView);
```

Comment:
```suggestion
    // inflate MapView and the LoadingView from layout
    mMapView = findViewById(R.id.mapView);
    mLoadingView = findViewById(R.id.loadingView);
```
Initialize the LoadingView and MapView just to keep code easier to read